pkg/sfu/streamallocator/streamallocator.go

```diff
@@ -555,7 +555,7 @@ func (s *StreamAllocator) postEvent(event Event) {
 	select {
 	case s.eventCh <- event:
 	default:
-		s.params.Logger.Warnw("stream allocator: event queue full", nil)
+		s.params.Logger.Warnw("stream allocator: event queue full", nil, "event", event.String())
```

Comment:
Seeing a few session where this queue gets hammered (guessing it is completely stuck). See like 20 - 30K logs for a participant at times. Adding event to understand a bit more.